jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamDatabasesResultSet.java

```diff
@@ -16,92 +16,21 @@
 package software.amazon.timestream.jdbc;
 
 import com.amazonaws.services.timestreamquery.model.ColumnInfo;
```

Comment:
For changes this file, I mostly just swapped the contents of this file with `TimestreamSchemasResultSet.java` and made it an empty resultset